packages/server/src/routes/abort.ts

```diff
@@ -1,8 +1,7 @@
 import { ResponseToolkit } from "@hapi/hapi";
-import { AbortRequest } from "@celo/payments-types";
 import { get } from "../storage";
 
-export function abort({ params }: AbortRequest, res: ResponseToolkit) {
+export function abort({ params }: any, res: ResponseToolkit) {
```

Comment:
A lot of the abort stuff is gone or commented out with this PR, but I assume we'll want to add it back in. I'm curious why this was switched to `any`.

Reply:
I just restored it with the new definition of the abort command, using the new interfaces as defined in the spec